test/integration/integration_test_utils.py

```diff
@@ -52,7 +40,16 @@ def aws_encryption_cli_is_findable():
 @pytest.fixture
 def cmk_arn():
     """Retrieves the target CMK ARN from environment variable."""
-    return os.environ.get(AWS_KMS_KEY_ID)
+    arn = os.environ.get(AWS_KMS_KEY_ID, None)
+    if arn is None:
+        raise ValueError(
+            'Environment variable "{}" must be set to a valid KMS CMK ARN for integration tests to run'.format(
+                AWS_KMS_KEY_ID
+            )
+        )
+    if 'key' not in arn:
```

Comment:
I think you want to ensure that this is both an ARN and not an alias.  To do that you should check to see if it starts with "arn:" and does *not* contain "alias"